bin/pull-kinto-admin.sh

```diff
@@ -5,7 +5,7 @@ VERSION=$(cat `pwd`/kinto-admin/VERSION)
 TAG="v${VERSION}"
 
 # download and unzip release
-curl -OL https://github.com/Kinto/kinto-admin/releases/download/${TAG}/kinto-admin-release.tar
```

Comment:
Can we avoid switching to `wget` by adding the `--fail` flag?

Reply:
Switched to alpine:3 but still using wget because it's included by default. Let me know if we're attached to curl though.

Reply:
Not attached to curl as much as not having to install another dependency. If alpine comes with `wget`, that works for me 👍🏻